### ISSUE TITLE
Rate limit hit

### ISSUE THREAD

**Issue by kevinbsc** [digitalmethodsinitiative/dmi-tcat#83]
I am getting this error message on DMI-TCAT query manager.

"Your latest rate limit hit was on 2014-11-23 22:11:33" 

What action is needed to resolve this issue? Thanks!

**Comment by dormeir99o** [comment]
I'm having the same problem, the rate limit message is just stuck there...

**Comment by ErikBorra** [comment]
Good to know, I'll reopen this issue so that we can modify the UI and e.g. allow users to click the message away.

**Comment by ErikBorra** [comment]
This could be picked up when working on #150

**Comment by dentoir** [comment]
I'm assuming the timestamp is not changing on the UI? In that case we simply have to define a window after which the message dissapears (a week, a month?).

**Comment by bacilo** [comment]
Hi! I just install tcat on a personal server (I've been using it on a shared server before) and I added 4 queries to test it out. Those were rather broad queries just to see if it was working.

I stopped them immediately, but the rate limit continues to show up (i.e. "Your latest rate limit was hit" with the current time, and the table tcat_error_ratelimit is continuously updated with tweets registered growing by millions an hour, even though all the queries are now stopped.

I'm guessing there is something going wrong with stopping the queries? Maybe I installed something wrong, but I don't see anything on the logs... grateful for any help!
